### PR TITLE
Add 2.2 config

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: configs/foreman/2.2.yaml [hunk 1]
```diff
@@ -0,0 +1,594 @@
...
+  - foreman-plugins-2.2-rhel7-dist
+  - foreman-plugins-2.2-el8
+  - foreman-client-2.2-el8
+  - foreman-client-2.2-rhel7
+  - foreman-client-2.2-rhel6
+  - foreman-client-2.2-rhel5
+  - foreman-client-2.2-sles11
+  - foreman-client-2.2-sles12
+:strict_keys: true
+:gpg_key: B6F08CCF
```
Comment (tbrisker) [inline configs/foreman/2.2.yaml:19]: This is the 2.1 key (also in other places below)

**Comment by ehelms** [comment]
Updated with the 2.2 key (double check me)

File: mash_scripts/foreman/2.2.0/foreman-2.2-el8.mash [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-2.2-el8
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-2.2-el8.mash:11]: this is still 2.1 key

File: mash_scripts/foreman/2.2.0/foreman-2.2-rhel7-dist.mash [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-2.2-rhel7-dist
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-2.2-rhel7-dist.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-client-2.2-el8.mash [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-client-2.2-el8
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-client-2.2-el8.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-client-2.2-rhel5.mash [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-client-2.2-rhel5
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-client-2.2-rhel5.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-client-2.2-rhel6.mash [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-client-2.2-rhel6
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-client-2.2-rhel6.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-client-2.2-rhel7.mash [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-client-2.2-rhel7
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-client-2.2-rhel7.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-client-2.2-sles11.mash [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-client-2.2-sles11
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-client-2.2-sles11.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-client-2.2-sles12.mash [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-client-2.2-sles12
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-client-2.2-sles12.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-plugins-2.2-el8.mash [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-plugins-2.2-el8
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-plugins-2.2-el8.mash:11]: ditto

File: mash_scripts/foreman/2.2.0/foreman-plugins-2.2-rhel7-dist.mash [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-plugins-2.2-rhel7-dist
+inherit = True
+strict_keys = True
+keys = B6F08CCF
```
Comment (tbrisker) [inline mash_scripts/foreman/2.2.0/foreman-plugins-2.2-rhel7-dist.mash:11]: ditto

**Comment by ehelms** [comment]
My sense of direction is bad today. Re-generated the configs.

**Pull request merged**